pkg/server/server.go

```diff
@@ -66,9 +66,11 @@ func (s *Server) createRouters() *negroni.Negroni {
 
 	n.UseHandler(r)
 
-	r.HandleFunc("/data/{deviceId}", dataController.GetAll).Methods("GET")
-	r.HandleFunc("/data/{deviceId}/sensor/{id}", dataController.GetByID).Methods("GET")
+	r.HandleFunc("/data/{user}", dataController.GetAll).Methods("GET")
+	r.HandleFunc("/data/{user}/sensor/{id}", dataController.GetByID).Methods("GET")
 	r.HandleFunc("/data", dataController.Create).Methods("POST")
+	r.HandleFunc("/data/{user}", dataController.DeleteAll).Methods("DELETE")
```

Comment:
As I commented above, you should break this commit up so that the endpoints could be added separately.